addon/templates/components/bootstrap/tables/-body.hbs

```diff
@@ -3,6 +3,7 @@
     {{#if hasBlock}}
       {{yield (hash
         rowIndex=index
+        rowData=rowData
```

Comment:
Is this supposed to be `rowsData`? https://github.com/wildland/ember-bootstrap-controls/blob/a12cb4086e1d2a18eec825a11e317471942ccba1/addon/components/bootstrap/tables/-body.js#L22

Reply:
Nope. This is the individual rowData from the `#each` so that you can use in to style a row based on the data.